src/cosmos-keystore.ts

```diff
@@ -86,13 +86,14 @@ function loadFromStorage(address: string): StoredWallet | null {
 }
 
 // stores an encrypted wallet in localstorage
-function addToStorage(name: string, address: string, ciphertext: string): void {
+function addToStorage(name: string, address: string, ciphertext: string, network?: string): void {
```

Comment:
should this be optional? I would propose we make this mandatory to keep the API clean.

Reply:
I think as we have this module in npm we should add backward compatibility to our code.

Reply:
Aren't for this case semver versions? The API changes so we bump the minor version?

Reply:
Updated)